msteams-platform/m365-apps/teamsjs-support-m365.md

```diff
@@ -0,0 +1,2069 @@
+---
+title: Teams JavaScript client library support across Microsoft 365
+author: erikadoyle
+ms.author: edoyle
+description: Understand the level of support for different TeamsJS library capabilities running in different hosts for Teams apps, including Microsoft Teams, Outlook, and Microsoft 365 app 
+ms.localizationpriority: high
+ms.topic: conceptual
+keywords: TeamsJS Teams JavaScript library capability Microsoft 365 M365
+---
+# TeamsJS capability support across Microsoft 365
+
+Starting with version 2.0.0, Teams JavaScript client library (TeamsJS) enables [certain types of Teams apps](./overview.md) to run across the Microsoft 365 ecosystem. Currently, Microsoft 365 application hosts (Microsoft 365 app and Outlook) for Teams apps support a subset of the application types and capabilities you can build for the Teams platform. This support will expand over time.
+
+This article details the level of support of Teams JavaScript client (TeamsJS) library version 2.x capabilities for *personal scope* tab apps and dialogs (task modules) across various Microsoft 365 host applications. For a more info on what's changed between TeamsJS versions 1.x and 2.x, see [What's new in TeamsJS version 2.x.x](../tabs/how-to/using-teams-client-library.md#whats-new-in-teamsjs-version-2xx).
```

Comment:
Is this true "personal scope" only because I thought that some of these were called in the Channel Tab/Chat Tab which are all contextual tabs. I think this should be changed

Reply:
Not sure I follow what the proposed change should be...do you mean removing the disclaimer that these tables only apply to personal scope tabs?